lfs/upload_test.go

```diff
@@ -923,3 +923,11 @@ func TestUploadVerifyError(t *testing.T) {
 		t.Errorf("verify not called")
 	}
 }
+
+func uploadMediaPath(t *testing.T, oid string) string {
```

Comment:
The other benefit of this approach is that we can see which tests are skipped at a glance.